examples/README.md

```diff
@@ -82,3 +82,9 @@ Learn more about annotation types in the [docs](https://docs.labelbox.com/docs/a
 | Model Diagnostics Demo  | [Github](model_diagnostics/model_diagnostics_demo.ipynb) | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/Labelbox/labelbox-python/blob/develop/examples/model_diagnostics/model_diagnostics_demo.ipynb) |
 | Custom Metrics Basics  | [Github](model_diagnostics/custom_metrics_basics.ipynb) | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/Labelbox/labelbox-python/blob/develop/examples/model_diagnostics/custom_metrics_basics.ipynb) | [Docs](https://docs.labelbox.com/docs/model-diagnostics) |
 | Custom Metrics Demo  | [Github](model_diagnostics/custom_metrics_demo.ipynb) | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/Labelbox/labelbox-python/blob/develop/examples/model_diagnostics/custom_metrics_demo.ipynb) |
+
+
+## [Extras](extras)
+| Notebook                        | Github                                         | Google Colab | Learn more |
+|---------------------------------|------------------------------------------------| ------------ | ---------- |
+| Classification Confusion Matrix | [Github](extras/classification-confusion-matrix.ipynb) | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/Labelbox/labelbox-python/blob/develop/examples/extras/classification-confusion-matrix.ipynb) | |
```

Comment:
Coming up with a `Not Found` error when trying to access. This is after pulling this branch as well. Am I doing something wrong to get the notebook?

```Fetch for https://api.github.com/repos/Labelbox/labelbox-python/contents/examples/extras?per_page=100&ref=develop failed: {
  "message": "Not Found",
  "documentation_url": "https://docs.github.com/rest/reference/repos#get-repository-content"
}```

Reply:
That's weird, it works for me on github. Is it supposed to work locally?